dockstore-client/src/main/java/io/dockstore/client/cli/nested/ToolClient.java

```diff
@@ -290,6 +303,24 @@ protected void handleListNonpublishedEntries() {
         }
     }
 
+    @Override
+    protected void handleListUnstarredEntries() {
+        try {
+            // check user info after usage so that users can get usage without live webservice
```

Comment:
This comment doesn't seem to make any sense here. Maybe it did before refactoring?